account_financial_report_qweb/security/account_financial_report_qweb.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Sergio Teruel - Tecnativa <sergio.teruel@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record id="group_one2many_financial_report_account" model="res.groups">
+    <field name="name">Filter financial reports accounts in One2many field</field>
```

Comment:
Objections here:
1. Why do you need a security group for this? I can see that it would be useful if you decide to activate it in the 'Accounting / Settings'. I can understand that some companies might want to keep this option invisible. But having to set it for each accounting user manually is, to me, not straightforward. 
1. I think that this name is very technical.  I think that it is easier as "Allow filtering on OCA financial reports by accounts"